docs/docsite/rst/rules/default_rules.rst

```diff
@@ -45,6 +45,7 @@ ID
 `E503 <https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/UseHandlerRatherThanWhenChangedRule.py>`_                                                                                                                                                                                   historic                                                                                                                                                                                                                                                                                                     Tasks that run when changed should likely be handlers                                                                                                                                                                                                                                                        If a task has a ``when: result.changed`` setting, it is effectively acting as a handler
 `E504 <https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/TaskNoLocalAction.py>`_                                                                                                                                                                                                     v4.0.0                                                                                                                                                                                                                                                                                                       Do not use 'local_action', use 'delegate_to: localhost'                                                                                                                                                                                                                                                      Do not use ``local_action``, use ``delegate_to: localhost``
+`E505 <https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/IncludeMissingFileRule.py>`_                                                                                                                                                                                                v4.3.0                                                                                                                                                                                                                                                                                                       referenced files must exist                                                                                                                                                                                                                                                                                  All files referenced by by include / import tasksmust exist. The check excludes files with jinja2 templates in the filename.
```

Comment:
`taskmust` missed a space. that file is generated but you need to sure is still ok.